theano/gpuarray/opt.py

```diff
@@ -346,20 +339,20 @@ def apply(self, fgraph):
             new_ops = None
             outputs = []
             # Apply the lifter
-            for lopt in (self.local_optimizers_map.get(node.op, []) +
-                         self.local_optimizers_map.get(type(node.op), []) +
-                         self.local_optimizers_all):
-                if move_to_GPU:
-                    t_opt = time.time()
-                    new_ops = lopt.transform(node.op, context_name,
-                                             [mapping[i] for i in node.inputs],
-                                             node.outputs)
-                    t_opt2 = time.time()
-                    time_opts[lopt] += t_opt2 - t_opt
-
-                    if new_ops:
-                        process_count[lopt] += 1
-                        break
+            if move_to_GPU:
+                for lopt in (self.local_optimizers_map.get(node.op, []) +
+                             self.local_optimizers_map.get(type(node.op), []) +
+                             self.local_optimizers_all):
+                        t_opt = time.time()
```

Comment:
no as this is a multiple of 4. It check for multiple of 4 and not 4 exactly from memory.